fcgi.c

```diff
@@ -174,7 +174,7 @@ handle_http (void *arg)
 {
     FCGX_Request *request = (FCGX_Request *) arg;
     char *rpath, *path, *length, *if_match, *if_none_match, *if_modified_since, *if_unmodified_since;
-    char *server_name, *server_port;
+    char *server_name, *server_port, *remote_addr, *remote_user;
```

Comment:
remote_port (REMOTE_USER is not a CGI environment variable)

Reply:
REMOTE_USER is part of the CGI 1.1 standard, and supported in some fcgi implementations. If REMOTE_USER is not defined in the env block the remote_user will be set to "unknown"